src/capacity_types.h

```diff
@@ -0,0 +1,16 @@
+#ifndef CYCLUS_SRC_CAPACITY_TYPES_H_
+#define CYCLUS_SRC_CAPACITY_TYPES_H_
+
+namespace cyclus {
+
+/// the type of capacity, currently less-than-or-equal-to (LTEQ) and
+/// greater-than-or-equal-to (GTEQ) are supported
+enum CapType {
+  LTEQ = 0,
+  GTEQ,
```

Comment:
I think that the standard abbreviations are GE and LE.  That would be more consistent with the same names they are given in the query filters.